.travis.yml

```diff
@@ -38,10 +25,42 @@ before_install:
 install:
   - yarn install --no-lockfile --non-interactive
 
-script:
-  - yarn lint:js
-  # We need to cleanup, since our deploy depends on the git repo's status
-  - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO
+# script:
+#   - yarn lint:js
+#   # We need to cleanup, since our deploy depends on the git repo's status
+#   - node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO
+
+notifications:
+  email: false
+
+stages:
+  - test
+  - additional tests
+  - versioned tests
+  - deploy
+
+jobs:
+  fail_fast: true
+
+  include:
+    - stage: test
+      env: NAME=test
+      script: yarn test
```

Comment:
Because the `test` script aliases to `ember try:each`, I think we're running every ember-try scenario in every job: https://travis-ci.org/ember-learn/ember-cli-addon-docs

Do want to make `yarn test` point to `ember try:one $EMBER_TRY_SCENARIO`, or does the Travis config need to change?

Reply:
Looking at ember-data-storefront, I think it has the opposite problem—it looks like it's always running with the default set of dependencies, regardless of the `EMBER_TRY_SCENARIO` variable.

Reply:
I think we had Storefront copy Ember Data after talking to Robert. I'm poking around Storefront's travis logs but can't see what versions of things it's using. Is there a way to have travis echo out the versions of everything yarn installed?

Reply:
I don't know of an easy way to do that, but normally ember-try has some pretty verbose output when it's doing its installation (plus a version grid at the end), and I don't see any of that there.